lib/tasks/metrics.rake

```diff
@@ -11,22 +11,29 @@ namespace :metrics do
       namespace.constants.map { |c| namespace.const_get(c) if namespace.const_get(c).is_a? Class }.compact
     }.flatten
 
+    SUPPLIERS = Supplier.all.to_a << nil
```

Comment:
What's the purpose of including an extra `nil` supplier in the array? Is that to generate aggregate results for all suppliers? If so might be worth a comment to mention that.

Reply:
Note added 👍